contracts/amm/MinterAmm.sol

```diff
@@ -118,6 +118,11 @@ contract MinterAmm is
     /// removing series)
     EnumerableSet.UintSet private openSeries;
 
+    uint256 private constant _NOT_ENTERED = 1;
```

Comment:
note, this may at first be alarming that I'm adding contract variables in between existing variables. This is intended because `maxOptionFeeBasisPoints`, `feeDestinationAddress`, and `ammDataProvider` do not exist on any deployed MinterAmm contracts. Instead you should look at the [v2](https://github.com/sirenmarkets/core/blob/v2/contracts/amm/MinterAmm.sol#L112) branch which is the logic we currently have deployed to existing AMMs.